config/webpack.config.prod.js

```diff
@@ -131,6 +111,13 @@ module.exports = {
         screw_ie8: true
       }
     }),
-    new ExtractTextPlugin('[name].[contenthash].css')
+    new ExtractTextPlugin('[name].[contenthash].css'),
+    function() {
```

Comment:
Here is the "manual" part of passing file names to static file which can then be loaded on the server.